packages/gvim.rb

```diff
@@ -71,5 +71,15 @@ def self.postinstall
     puts "If you are upgrading from an earlier version, edit ~/.bashrc".orange
     puts "and remove the 'export VIMRUNTIME' and 'export LC_ALL=C' lines.".orange
     puts
+    puts "If you receive the following error while attempting to launch gvim:".orange
+    puts
+    puts "E233: cannot open display"
+    puts "Press ENTER or type command to continue"
+    puts "E852: The child process failed to start the GUI"
+    puts "Press ENTER or type command to continue"
+    puts
+    puts "Execute the following:".orange
+    puts "echo 'export GDK_BACKEND=x11' >> ~/.bashrc && source ~/.bashrc".orange
```

Comment:
Perhaps we should change this in `sommelier` instead of overwriting the variable in every package that it breaks?

Reply:
That is a good thought.  This fix is needed for code/codium also.